Packaging.Targets/Packaging.Targets.csproj

```diff
@@ -22,6 +22,12 @@
   <PropertyGroup Condition="'$(Configuration)|$(TargetFramework)|$(Platform)'=='Debug|netstandard1.5|AnyCPU'">
     <PlatformTarget>AnyCPU</PlatformTarget>
   </PropertyGroup>
+  <!-- Copy build task assemby to build directory -->
+  <PropertyGroup>
+    <BuildOutputTargetFolder>tools</BuildOutputTargetFolder>
+    <!-- Suppress warning about lib being empty -->
+    <NoPackageAnalysis>true</NoPackageAnalysis>
```

Comment:
Can we suppress that individual warning instead? Will users who add a reference to Packaging.Targets get a warning?

Reply:
No idea, quick googling gave me no results.
No, there won't be any warnings for people who just reference the package.

Reply:
OK, if this doesn't impact the end users, it's fine with me 😄